### PR TITLE
Creating Footer

### PR DESCRIPTION
**Pull request by YamanSammani**
Creating the website Footer.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,11 +3,17 @@
     "version": "0.1.0",
     "private": true,
     "dependencies": {
+<<<<<<< HEAD
+        "@fortawesome/fontawesome-svg-core": "^1.2.32",
+        "@fortawesome/free-brands-svg-icons": "^5.15.1",
+        "@fortawesome/react-fontawesome": "^0.1.12",
+=======
         "@ant-design/icons": "^4.2.2",
+>>>>>>> bbb3a84ae05d72eb85c4e13d84925739ce6eb13d
```
Comment (tmansuroglu) [inline package.json:12]: leftover from merge

**Review by tmansuroglu — CHANGES_REQUESTED** [review]
There is already a PR for footer. You should delete 1 of them.

**Comment by louisrli** [comment]
yaman can you please fix this by sunday latest. all of the PRs from the
first sprint absolutely need to be in by then. and i mean fixed fixed, sit
down with your teammates in a 1 hour meeting if you need to.

**Comment by YamanSammani** [comment]
This PR will be closed since a new PR was made for the footer, it called Footer updated.

**Pull request closed without merging**